openreview/venue/matching.py

```diff
@@ -250,12 +251,14 @@ def _build_note_conflicts(self, submissions, user_profiles, get_profile_info, co
 
         ## for AC conflicts, check SAC conflicts too
         sac_user_info_by_id = {}
-        sac_assignment_edges = {}
         if self.is_area_chair:
             sacs_by_ac =  { g['id']['head']: [v['tail'] for v in g['values']] for g in self.client.get_grouped_edges(invitation=self.venue.get_assignment_id(self.venue.get_senior_area_chairs_id()), groupby='head', select=None)}
             if sacs_by_ac:
                 sac_user_profiles = openreview.tools.get_profiles(self.client, self.client.get_group(self.venue.get_senior_area_chairs_id()).members, with_publications=True)
-                sac_user_info_by_id = { p.id: info_function(p, compute_conflicts_n_years) for p in sac_user_profiles }
+                if self.sac_profile_info:
+                    sac_user_info_by_id = { p.id: self.sac_profile_info(p, self.venue.get_submission_venue_id()) for p in sac_user_profiles }
```

Comment:
I would like all the profile_info functions to have the same signatures so they can be used everywhere conflicts need to be computed.

Reply:
we could use the module `inspect` to check if the profile_info function expects `submission_venue_id` as a parameter. 

Example from the chat:

 
<img width="753" alt="Screen Shot 2023-05-08 at 10 48 49 AM" src="https://user-images.githubusercontent.com/545506/236855982-abe7e6d5-d5a7-4841-9870-c0c95fc1d7c8.png">